sid/simulate.py

```diff
@@ -87,29 +93,56 @@ def simulate(
         states, assort_bys, params, contact_models
     )
 
-    to_concat = []
     for period, date in enumerate(duration["dates"]):
         states["date"] = date
-        states["period"] = period
+        states["period"] = np.uint16(period)
 
         contacts = calculate_contacts(
             contact_models, contact_policies, states, params, date
         )
-        infections, states = calculate_infections(
+        newly_infected, states = calculate_infections(
             states, contacts, params, indexers, first_probs, seed,
         )
-        states = update_states(states, infections, params, seed)
+        states = update_states(states, newly_infected, params, seed)
 
         for i, contact_model in enumerate(first_probs):
             states[contact_model] = contacts[:, i]
-        states["infections"] = infections
-        to_concat.append(states.copy(deep=True))
+        states["infections"] = newly_infected
```

Comment:
```suggestion
        states["newly_infected"] = newly_infected
```

Reply:
Ok, I have to admit I was lazy. I found the internal name not descriptive enough, but I did not want to break other code which works with this specific column name like the plots.

If we change this, let us discuss quickly other column names and change them if they are not descriptive enough.

Reply:
So far this column isn't used anywhere, including the plots. 

Here are the columns in the output of the tutorial notebooks plus my suggestions: 

index, date and background:
- index, date, period, age_group, region

health states: 
- ever_infected, immune, infectious, knows, needs_icu, dead
- **symptoms** -> symptomatic
- **infections** -> newly_infected

counter: 
- cd_infectious_false
- **infection_counter** -> nr_infected

contact model specific variables: 
- **close** -> nr_infected_close
- **distant** -> nr_infected_distant
- missed_close
- missed_distant 

@janosg?

Reply:
What is **infection_counter** again?

I would choose
- **close** -> n_contacts_close
- missed_close -> n_missed_contacts_close

Reply:
n_contacts_close etc are good. The infection_counter counts how many people a person infected during her most recent or current infection with the virus. This is used to calculate r_zero.

Reply:
Maybe `n_has_infected`?